examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/dem-packing-in-spouted-bed-generator.prm

```diff
@@ -0,0 +1,133 @@
+    set threshold               = 0.5
+    set dynamic check frequency = 10000
+  end
+  set particle particle contact force method = hertz_mindlin_limit_overlap
+  set particle wall contact force method     = nonlinear
+  set integration method                     = velocity_verlet
+end
+
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+
+subsection lagrangian physical properties
+  set gx                       = 0
+  set gy                       = 0
+  set gz                       = -9.81
+  set number of particle types = 1
+  subsection particle type 0
+    set size distribution type            = uniform
+    set diameter                          = 0.005
+    set number of particles               = 100000
+    set density particles                 = 100 #2526 
+    set young modulus particles           = 1e6
+    set poisson ratio particles           = 0.25
+    set restitution coefficient particles = 0.97
+    set friction coefficient particles    = 0.4
+    set rolling friction particles        = 0.3
+  end
+  set young modulus wall           = 1e6
+  set poisson ratio wall           = 0.25
+  set restitution coefficient wall = 0.33
+  set friction coefficient wall    = 0.2
+  set rolling friction wall        = 0.3
+end
+
+#---------------------------------------------------
+# Insertion Info
+#---------------------------------------------------
+
+subsection insertion info
+  set insertion method                               = volume
+  set inserted number of particles at each time step = 100000
+  set insertion frequency                            = 2000
+  set insertion box minimum x                        = -0.075
+  set insertion box minimum y                        = -0.075
+  set insertion box minimum z                        = 0
+  set insertion box maximum x                        = 0.075
+  set insertion box maximum y                        = 0.075
+  set insertion box maximum z                        = 0.7
+  set insertion distance threshold                   = 1.05
+  set insertion maximum offset                       = 0.3
+  set insertion prn seed                             = 19
+end
+
+#---------------------------------------------------
+# Floating walls
+#---------------------------------------------------
+
+subsection floating walls
+  set number of floating walls = 1
+  subsection wall 0
+    subsection point on wall
+      set x = 0
+      set y = 0
+      set z = 0
+    end
+    subsection normal vector
+      set nx = 0
+      set ny = 0
+      set nz = 1
+    end
+    set start time = 0
+    set end time   = 50
+  end
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+
+subsection mesh
+  set type                                = gmsh
+  set file name                           = ./mesh/test_0403.msh
+  set expand particle-wall contact search = false
```

Comment:
This should be set to true no since the geoemtry is cylindrical.
You can read it in the documentation here: https://lethe-cfd.github.io/lethe/documentation/parameters/dem/mesh.html
It is quite well explained.
```suggestion
  set expand particle-wall contact search = true
```